var/spack/repos/builtin/packages/python/package.py

```diff
@@ -203,8 +203,11 @@ def python_ignore(self, ext_pkg, args):
                 r'site-packages/site[^/]*\.pyc?$',
                 r'site-packages/__pycache__/site[^/]*\.pyc?$'
             ])
+        if ext_pkg.name != 'py-pygments':
+            patterns.append(r'bin/pygmentize$')
```

Comment:
Does this affect multiple packages? I'm wondering if this should be in the package itself. Or maybe all of the `ignore=*` things in each package should just be moved here someday. Thoughts?